tool/tsh/tsh.go

```diff
@@ -864,6 +868,10 @@ func onLogin(cf *CLIConf) error {
 				return trace.Wrap(err)
 			}
 			printProfiles(cf.Debug, profile, profiles)
+			_, err := tc.PingAndShowMOTD(cf.Context)
```

Comment:
Hmm, you already have PingAndShowMOTD above, aren't we going to display it twice if you also put it in each "case" branch here?

Reply:
Ah yeah, I meant to remove that, it could also print twice being at the top if it flows through too.

Reply:
I did it like this to retain the previous `tc.Login` behavior -- it shows the MOTD when its called and uses the result of the ping. I can change it so showing the MOTD is done separatly and the ping is passed in

Reply:
@lxea Ah yes, it makes sense. In that case it should be fine.

Reply:
But can you please move it so MOTD is displayed before everything else like profiles?